Lib/test/_test_multiprocessing.py

```diff
@@ -2694,6 +2694,10 @@ def common(self, manager):
 
         self.assertEqual(foo.f(), 'f()')
         self.assertRaises(ValueError, foo.g)
+        with self.assertRaises(ValueError) as ctx:
+            foo._callmethod('g')
```

Comment:
Why don't you simply call `foo.g` here? Is `_callmethod` doing something special?

Reply:
I don't remember, really, but `foo.g()` seems to work as well, so I'll change it